right-for-life_front/src/app/happy-stories-list/happyStoriesList.js

```diff
@@ -0,0 +1,35 @@
+import React, { useState, useEffect } from 'react';
+import { HappyStoriesListItem } from '../happy-stories-list-item';
```

Comment:
do not named folder with -, use CamelCase instead

Reply:
Got it

Reply:
Should I fix it now?

Reply:
yeap! fix-commit-push, it's natural lifecycle:)